### ISSUE TITLE
Comms Issue #23

### ISSUE THREAD

**Issue by coleweusthem** [asc-csa/Scisat-App#55]
For the form validation, you must use the WET-BOEW plugin https://wet-boew.github.io/v4.0-ci/docs/ref/formvalid/formvalid-en.html

**Comment by nfee006** [comment]
It's not clear if this plugin will even work since these aren't technically HTML form elements. Will happily be proven wrong.

**Comment by coleweusthem** [comment]
@nfee006, I didn't create a label for dash issues in scisat yet. I had identified this one as being an issue in my email. I'll flag it now

**Comment by coleweusthem** [comment]
@nfee006, I'm going to start working on this, like you said it might not apply as these are filters not forms. If you hear different from comms let me know

**Comment by coleweusthem** [comment]
@nfee006, I did some experimenting (The new GCWeb theme doesn't have up to date documentation) and it looks like we would need to wrap all the filters in a form to make this work, as well as some other modifications. It may be worth double checking with comms this is what is required before investing more time into it.

**Comment by nfee006** [comment]
@coleweusthem Roger that, on it.
